src/ui/components/DismissibleTextForm/index.js

```diff
@@ -93,16 +131,31 @@ export class DismissibleTextFormBase extends React.Component<
     invariant(onDismiss, 'onDismiss() is required');
 
     onDismiss();
+    this.localState.clear();
```

Comment:
I see that dismissing the form does not reset its value when re-opened. Is that intended? The value gets removed from the localstorage, but it is still present in the `textarea`.

Reply:
I think it was implemented that way but I can make it clear. I reviewed all usage of `onDismiss` and I think it would be an intuitive feature.